vm/os-windows.cpp

```diff
@@ -304,6 +304,54 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+static volatile bool stop_on_ctrl_break = false;
+static HANDLE ctrl_break_thread = NULL;
+
```

Comment:
Global variables are frowned upon. Instead place `ctrl_break_thread` in `factor_vm`. Like how the `sampler_thread` is handled.